packages/client/hmi-client/src/components/model/model-parts/tera-model-part-entry.vue

```diff
@@ -90,25 +91,29 @@ const props = defineProps<{
 
 const emit = defineEmits(['update-item']);
 
+const debouncer = debounce((key: string, value: string = '') => {
+	emit('update-item', { key, value });
+}, 300);
+
 const nameText = computed({
 	get: () => props.name,
-	set: (newName) => emit('update-item', { key: 'name', value: newName })
+	set: (newName) => debouncer('name', newName)
 });
 const unitExpression = computed({
 	get: () => props.unitExpression,
-	set: (newUnitExpression) => emit('update-item', { key: 'unitExpression', value: newUnitExpression })
+	set: (newUnitExpression) => debouncer('unitExpression', newUnitExpression)
```

Comment:
I don't think this works, because the later debouncer will replace the earlier debouncers, so would that mean you just cancel your prior updates?

e.g `unitExpression` will override earlier `grounding`

Reply:
I didn't see that during my testing.

Reply:
We chatted. This is not an ideal solution but suffice for the time being. We can merge and do a follow up to make this mechanic more robust.